test/Dialect/FIRRTL/emit-metadata.mlir

```diff
@@ -91,6 +91,9 @@ firrtl.circuit "BasicBlackboxes" attributes { annotations = [{
   firrtl.extmodule @ignored5() attributes {annotations = [{class = "sifive.enterprise.grandcentral.transforms.SignalMappingAnnotation"}], defname = "ignored5"}
   firrtl.extmodule @ignored6() attributes {annotations = [{class = "firrtl.transforms.BlackBox"}], defname = "ignored6"}
 
+  // ScalaClassAnnotation should be discarded after this pass.
+  // CHECK: firrtl.extmodule @ignored2() attributes {defname = "ignored2"}
```

Comment:
Nit: for explicit negative checking, consider the following:

```
CHECK:     firrtl.extmodule @ignored2()
CHECK-NOT:   sifive.enterprise.firrtl.ScalaClassAnnotation
```

Only for your consideration. The comment w/ the CHECK clarifies it fine.

Reply:
This is better for sure.